samples/cext-trey-research-python/.vscode/launch.json

```diff
@@ -0,0 +1,69 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Launch App (Edge)",
+            "type": "msedge",
+            "request": "launch",
+            "url": "https://teams.microsoft.com/l/app/${{local:TEAMS_APP_ID}}?installAppPackage=true&webjoin=true&${account-hint}",
```

Comment:
As this project contains a declarative agent, we should navigate to Microsoft 365 Copilot instead of Teams.

Reply:
Please provide us with the link to be added here

Reply:
https://www.microsoft365.com/chat

Reply:
fixed